src/Entity/Commande.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Entity;
+
+class Commande
+{
+    private $id;
+
+    private $numCommande;
+
+    private $elements;
+
+    private $user;
+
+    private $status;
+
+    public function __construct($numCommande, $elements, $user, $status)
+    {
+        $this->id = \App\ORM\Util\UUID::v4();
+        $this->numCommande = $numCommande;
```

Comment:
just my opinion, but the id could have been used as order number, I mean... it's unique so it already does the job.